test/app/components/Settings/scheduler/create-modal.test.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { mount, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+global.document.createRange = function() {
+    return {
+        setEnd: function() {},
+        setStart: function() {},
+        getBoundingClientRect: function() {
+            return { right: 0 };
+        },
+        getClientRects: function() {
+            return { length: 0 };
+        }
+    };
+};
+const CreateModal = require('../../../../../app/components/Settings/scheduler/create-modal.jsx').default;
```

Comment:
I'm not terribly familiar with `jest`. Don't we need to use [jest.unmock](https://jestjs.io/docs/en/jest-object#jestunmockmodulename)?


All the other tests use it; e.g. https://github.com/plotly/falcon-sql-client/blob/824b3920738f0f51d631aee2421aa9f1ae23135b/test/app/components/Settings/Preview/chart-editor.test.jsx#L10

Also, is there a reason for using `require` instead of `import`?

Reply:
There is no need to use `jest.unmock` in this test since we aren't mocking anything using `jest.mock`. If anything I should just `delete global.document.createRange` in the `afterAll` hook.

Also the reason for `require` instead of `import` is because `require` is run sequentially after `global.document.createRange = function() {` and `import` is hoisted to the top. Typically this doesn't matter either but CodeMirror has side-effects at the top level which is causing failures on import.